src/core/symbology-ng/qgsmarkersymbollayerv2.cpp

```diff
@@ -2152,20 +2173,63 @@ void QgsFontMarkerSymbolLayerV2::renderPoint( QPointF point, QgsSymbolV2RenderCo
   if ( !p )
     return;
 
-  QColor penColor = mColor;
+  QMatrix transform;
```

Comment:
QMatrix is obselete (see http://doc.qt.io/qt-4.8/qmatrix.html). Use QTransform instead.